test_twitter.py

```diff
@@ -9,22 +11,17 @@ def local_picture():
         return f.read()
 
 
-@pytest.fixture
-def picture_id(local_picture):
-    twitter = Twitter()
-    pic_id, pic_name = twitter.upload_photo("test", local_picture)
-    return pic_id
-
-
-@pytest.fixture
-def tweet_id():
-    twitter = Twitter()
-    tw_id = twitter.create_post("test")
-    return tw_id
+def upload_photo_mock_request(*args, **kwargs):
```

Comment:
This looks very non-intuitive from the test level. Just move appropriate config into the tests, without param checking.